roles/robot-pkgs/tasks/main.yml

```diff
@@ -25,7 +25,7 @@
     path: '{{ global_profile_path }}'
     marker: "## {mark} ROS profile entries ##"
     block: |
-      source /opt/ros/{{ ros_release }}/setup.bash
+      source {{ global_base_path }}/ros/{{ ros_release }}/setup.bash
```

Comment:
I'm not sure /opt should be templated here, since it comes from the apt package, not from Ansible.